opm/parser/eclipse/EclipseState/Schedule/OilVaporizationProperties.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef DRSDT_HPP
+#define DRSDT_HPP
+#include <iostream>
+#include <string>
+#include <memory>
+#include <opm/parser/eclipse/EclipseState/Schedule/ScheduleEnums.hpp>
+
+namespace Opm
+{
```

Comment:
I agree -- I do not really understand the relationship between the VAPPARS, DRVDT and DRSDT keywords enough to understand why this logically belongs in a single class, in particular why (or if?) behaviour is exclusive in some sense (since you can only construct one of them for a particular instance).